sujeong/app.js

```diff
@@ -45,12 +47,129 @@ app.post('/users',async (req ,res,next)=>{
             email,
             profile_image,
             password
-        ) VALUES ( ?, ?, ?, ?)
+        ) VALUES ( ?, ?, ?, ?);
         `,[name, email, profile_image, password]
     );
     res.status(201).json({message:"userCreated"})
 })
 
+
+app.post('/posts',async(req, res, next)=>{
+    const {title, content, user_id,image_url} =req.body
+    
+    await appDataSource.query(
+        `INSERT INTO posts(
+            title,
+            content,
+            user_id,
+            image_url
+        ) VALUES (?,?,?,?);`
+    ,[title, content, user_id, image_url]
+    );
+    res.status(201).json({message: "postCreated"})
+})
+
+
+app.get('/users_posts', async(req, res)=>{
+    
+    await appDataSource.query(
+    `SELECT
+    u.id as userId,
+    u.profile_image as userProfileImage,
+    p.user_id as postingId,
+    p.image_url as postingImageUrl,
+    p.content as postingContent
+    FROM users u
+    INNER JOIN posts p ON u.id = p.user_id;
+    `,(err, rows) => {
+        res.status(200).json(rows);
+    });
+    });
```

Comment:
수정님 ! 해당 GET 요청의 엔드포인트를 봤을 때, 어떠한 정보를 가져오는지 이해하기 어렵습니다
더불어 엔드포인트 설정시 '_' 언더바 사용을 자제해 주시고, 오전에 있는 RESTFUL API 세션을 들으신 다음 꼭 수정해 주시기 바랍니다 
- 해당하는 유저의 게시물을 모두 가져오는 엔드포인트의 경우
- `/posts/:userId`와 같이 엔드포인트만 봤을  때, 이게 어떤 역할을 하는 API인지 판단이 될 수 있게끔 고민해 보세요!
```suggestion
app.get('/posts/:userId', async(req, res)=>{
    
    await appDataSource.query(
    `SELECT
        u.id as userId,
        u.profile_image as userProfileImage,
        p.user_id as postingId,
        p.image_url as postingImageUrl,
        p.content as postingContent
    FROM users u
    INNER JOIN posts p ON u.id = p.user_id;
    `,(err, rows) => {
        res.status(200).json(rows);
    });
});
```